AK/Math.h

```diff
@@ -250,6 +250,10 @@ constexpr void sincos(T angle, T& sin_val, T& cos_val)
         "fsincos"
         : "=t"(cos_val), "=u"(sin_val)
         : "0"(angle));
+#elif ARCH(AARCH64)
+    sin_val = sin(angle);
+    cos_val = cos(angle);
+    return;
 #else
     __builtin_sincosf(angle, sin_val, cos_val);
 #endif
```

Comment:
This part can be removed completely. These builtins are architecture-independent, so if it doesn't work on ARM, there's a large chance that it won't work on anything else either.